LiveSplit.4DGolf.asl

```diff
@@ -85,18 +84,20 @@ init {
 
 		vars.Unity.Make<int>(gs.Static, gs["balls"]).Name = "balls"; // Array of balls. This gets created the moment you press play
 		vars.Unity.Make<bool>(gs.Static, gs["balls"], 0x20, b4["sinking"]).Name = "ballSinking"; // thearst3rd: where does 0x20 come from?
+		
+		vars.Unity.Make<int>(gs.Static, gs["balls"], 0x20).Name = "ballAddress"; // Is a non-zero number if the ball is loaded, and zero when it isn't
+		vars.Unity["ballAddress"].FailAction = (MemoryWatcher.ReadFailAction)1; // Configures the MemoryWatcher to return 0 if pointer fails a read. By default it returns the last read value.
```

Comment:
Does the following work?
```suggestion
		vars.Unity["ballAddress"].FailAction = MemoryWatcher.ReadFailAction.SetZeroOrNull; // Configures the MemoryWatcher to return 0 if pointer fails a read. By default it returns the last read value.
```
https://github.com/LiveSplit/LiveSplit/blob/f6921db78cfe1bf05a2ebd5b1af4408a348e4d12/src/LiveSplit.Core/ComponentUtil/MemoryWatcher.cs#L59